dev/projection/script-undo.js

```diff
@@ -83,7 +83,7 @@ Undo.createNode = (element, parent, options = {}, overrideId) => {
         scheduleRender()
     }
 
-    node.render = () => visualElement.syncRender()
+    node.render = () => visualElement.render()
```

Comment:
What's the difference between `syncRender` and `render`?

Reply:
I just renamed the method as there wasn't a public `render` method anyway and so the `sync` seemed a bit redundant, perhaps it was legacy